src/Tribe/Rewrite.php

```diff
@@ -487,11 +492,38 @@ public function remove_percent_placeholders( array $rules ) {
 
 			foreach ( $rules as $key => $value ) {
 				$key = str_replace( self::PERCENT_PLACEHOLDER, '%', $key );
-				$new_rules[$key] = $value;
+				$new_rules[ $key ] = $value;
 			}
 
 			return $new_rules;
 		}
+
+		/**
+		 * A function to Add an \ before the required caracters in Regex
+		 * Avoid using it elsewhere
+		 *
+		 * @param  string $value
+		 * @return string
+		 */
+		public function _build_replace_esc_regex( $value ) {
+			return '\\' . $value;
+		}
+
+		/**
+		 * Escapes the Required characters on a Regular Expression Word
+		 * Avoid using it elsewhere
+		 *
+		 * @param  string $base
+		 * @return string
+		 */
+		public function _esc_regex_elements( $base ) {
```

Comment:
Seems really close to what [preg_quote()](http://php.net/manual/en/function.preg-quote.php) does - is there a reason not to use that (can we add a note in the docblock if so)?